pmd-core/src/main/java/net/sourceforge/pmd/cpd/token/AntlrToken.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.cpd.token;
+
+import org.antlr.v4.runtime.Lexer;
+import org.antlr.v4.runtime.Token;
+
+import net.sourceforge.pmd.lang.ast.GenericToken;
+
+import com.beust.jcommander.internal.Nullable;
```

Comment:
refrain from using this annotation, as it's internal from a dependency used just for the CLI